src/s1reader/s1_reader.py

```diff
@@ -349,29 +374,46 @@ def burst_from_xml(annotation_path: str, orbit_path: str, tiff_path: str,
     # burst sensing starts in prototyping test data
     burst_interval = 2.758277
 
-    #parse manifest.safe to retrieve IPF version
+    # parse manifest.safe to retrieve IPF version
     manifest_path = os.path.dirname(annotation_path).replace('annotation','') + 'manifest.safe'
     with open_method(manifest_path, 'r') as f_manifest:
         tree_manfest = ET.parse(f_manifest)
         ipf_version = get_ipf_version(tree_manfest)
 
-    #Load the Product annotation - for EAP calibration
-    #with open_method(annotation_path, 'r') as f_lads:
-    #    tree_lads = ET.parse(f_lads)
-    #    product_annotation = s1_annotation.ProductAnnotation.from_et(tree_lads)
+    # Load the Product annotation - for EAP calibration
+    with open_method(annotation_path, 'r') as f_lads:
+        tree_lads = ET.parse(f_lads)
+        product_annotation = s1_annotation.ProductAnnotation.from_et(tree_lads)
 
-    #load the Calibraton annotation
-    calibration_annotation_path = annotation_path.replace('annotation/', 'annotation/calibration/calibration-')
+    # load the Calibraton annotation
+    calibration_annotation_path =\
+        annotation_path.replace('annotation/', 'annotation/calibration/calibration-')
     with open_method(calibration_annotation_path, 'r') as f_cads:
         tree_cads = ET.parse(f_cads)
         calibration_annotation = s1_annotation.CalibrationAnnotation.from_et(tree_cads)
 
-    #load the Noise annotation
+    # load the Noise annotation
     noise_annotation_path = annotation_path.replace('annotation/', 'annotation/calibration/noise-')
     with open_method(noise_annotation_path, 'r') as f_nads:
         tree_nads = ET.parse(f_nads)
         noise_annotation = s1_annotation.NoiseAnnotation.from_et(tree_nads, ipf_version)
 
+    # load AUX_CAL annotation
+    flag_apply_eap = is_eap_correction_necessary(ipf_version)
+    if flag_apply_eap.phase_correction:
+        path_aux_cal = get_path_aux_cal(f'{os.path.dirname(s1_annotation.__file__)}/data/aux_cal',
+                                        platform_id,
+                                        product_annotation.inst_config_id)
+        
+        with zipfile.ZipFile(path_aux_cal,'r') as zipfile_aux_cal:
+            str_safe_aux_cal = os.path.basename(path_aux_cal).replace('.zip','')
+            with zipfile_aux_cal.open(f'{str_safe_aux_cal}/data/s1a-aux-cal.xml','r') as f_aux_cal:
+                tree_aux_cal = ET.parse(f_aux_cal)
+        aux_cal_subswath = s1_annotation.AuxCal.from_et(tree_aux_cal, pol, subswath_id)
```

Comment:
What do you think about creating a `load_from_zip_file` classmethod in `AuxCal` that contains all the zip file handling? I think something like that would help declutter an already crowded `s1_reader.burst_from_xml`.

Not relevant to this PR: the same logic could be applied to all the other annotations readers above.

Reply:
I think that is a good idea. I've implemented `s1_annotation.AucCal.load_from_zip_file()`

>Not relevant to this PR: the same logic could be applied to all the other annotations readers above.
I'll keep that in mind by opening an issue tracker #67